assets/src/edit-story/components/panels/panel/handle.js

```diff
@@ -33,24 +33,34 @@ import useDragHandlers from './useDragHandlers';
 import useKeyboardHandlers from './useKeyboardHandlers';
 
 const Handle = styled.button.attrs( { type: 'button', role: 'separator' } )`
-	background: transparent;
+	background-color: ${ ( { theme } ) => theme.colors.fg.v5 };
 	border: 0;
 	padding: 0;
-	height: 10px;
+	height: 6px;
 	display: flex;
 	flex-direction: column;
-	justify-content: flex-start;
+	justify-content: center;
 	align-items: center;
 	cursor: row-resize;
+
+	&:focus,
+	&:active {
+		outline: none;
+	}
 `;
 
 const Bar = styled.div`
-	margin-top: 4px;
-	background-color: ${ ( { theme } ) => theme.colors.bg.v0 };
-	width: 32px;
+	background-color: ${ ( { theme } ) => theme.colors.bg.v1 };
```

Comment:
I thought it's `#000000` (and thus `v0`) on Figma 🤔

Reply:
It's `#000000` but with an opacity on Figma, confused me too

Reply:
But you have `opacity: 0.1;` further down as well, so that's `#191C28` with opacity

Reply:
You're right! Changed.